static/js/stream_bar.js

```diff
@@ -19,6 +20,9 @@ function update_compose_stream_icon(stream_name) {
     } else if (stream_data.is_web_public_by_stream_name(stream_name)) {
         $globe_icon.show();
         $streamfield.addClass("lock-padding");
+    } else {
+        $hashtag.show();
+        $streamfield.addClass("lock-padding");
```

Comment:
Since `$streamfield.addClass("lock-padding");` is common in these conditions, let's extract it above.